pandas/tests/frame/test_constructors.py

```diff
@@ -2780,6 +2780,18 @@ def test_construction_nan_value_timedelta64_dtype(self):
         )
         tm.assert_frame_equal(result, expected)
 
+    def test_dataframe_from_array_like_with_name_attribute(self):
+        class DummyArray(np.ndarray):
```

Comment:
Can you start this test with a comment referring to the GitHub issue

    # https://github.com/pandas-dev/pandas/pull/61451